README.md

```diff
@@ -1,128 +1,207 @@
 ## [简体中文](/README_CN.md)
-# thprac
-A tool for practicing Touhou Shmups
-## [Download](https://github.com/touhouworldcup/thprac/releases/latest) ([latest beta](https://nightly.link/touhouworldcup/thprac/workflows/main/master/thprac.zip))
-[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/V7V7O03J4)
 
-# How to use
-Either: place the executable in the same folder as your game, and run it. The game must be updated to the latest version and the executable must have it's original filename (thXX.exe for most games, 東方紅魔郷.exe for Embodment of Scarlet Devil, alcostg.exe for Uwabami Breakers). Thprac will automatically detect vpatch. Steam versions will not work with this method.
+# **thprac**
+> thprac is a tool for practicing Touhou shmups.
 
-Or: Run the game in whatever way you want (be it through thcrap, with vpatch, through Steam, etc...), then run the thprac exe. Thprac will automatically detect the running game and attach itself to it. You may need to return to the main menu for thprac to take effect.
+[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/V7V7O03J4)
 
-Or: Use the launcher. This video explains how the launcher works.
+## **Table of Contents**
+* [Downloading](#downloading)
+* [Usage](#usage)
+* [Compatability](#compatability)
+* [Features](#features)
+* [Backspace Menu](#backspace-menu)
+* [Advanced Options](#advanced-options)
+* [Statistics](#statistics)
+* [Game Specific Features](#game-specific-features)
+* [Credits](#credits)
+* [FAQ](#faq)
+
+## <a name="downloading"></a> **Downloading**
```

Comment:
I put this at the very very top because this GitHub page is essentially thprac's website. The problem is that people don't know how to use GitHub, so I want a direct download button as high up on the page as possible.